documentation-site/pages/blog/introducing-react-view/index.mdx

```diff
@@ -0,0 +1,278 @@
+import metadata from './metadata.json';
+import LiveEditor from './live-editor';
+import Layout from '../../../components/layout';
+import Yard from '../../../components/yard/index';
+import {BlogImage, Meta, Caption} from '../../../components/blog';
+
+import {View, PropTypes} from 'react-view';
+import {Button, SIZE} from 'baseui/button';
+
+export default Layout;
+
+<Meta data={metadata} />
+
+React View is an **interactive playground**, **documentation** and **code generator** for your components. You can use it to document your React component APIs, let users to live-edit the props and grab the generated ready-to-use source code.
```

Comment:
let users to live-edit the props...
->
enabling users to live-edit the props...